MapboxNavigation/NavigationViewController.swift

```diff
@@ -300,49 +308,30 @@ open class NavigationViewController: UIViewController {
     /**
      If `true`, `UIApplication.isIdleTimerDisabled` is set to `true` in `viewWillAppear(_:)` and `false` in `viewWillDisappear(_:)`. If your application manages the idle timer itself, set this property to `false`.
      */
-    @objc public var shouldManageApplicationIdleTimer = true
+    public var shouldManageApplicationIdleTimer = true
     
     /**
      Bool which should be set to true if a CarPlayNavigationView is also being used.
      */
-    @objc public var isUsedInConjunctionWithCarPlayWindow = false {
+    public var isUsedInConjunctionWithCarPlayWindow = false {
         didSet {
-            self.mapViewController?.isUsedInConjunctionWithCarPlayWindow = self.isUsedInConjunctionWithCarPlayWindow
-        }
-    }
-    
-    var isConnectedToCarPlay: Bool {
-        if #available(iOS 12.0, *) {
-            CarPlayManager.shared.isConnectedToCarPlay
-        } else {
-            false
+            self.mapViewController.isUsedInConjunctionWithCarPlayWindow = self.isUsedInConjunctionWithCarPlayWindow
```

Comment:
Now that you've made `mapViewController` non-optional, how about making this var proxy to the central underlying var, rather than copying the state around.

It seems like it would be easier to keep things in sync, e.g. if `mapViewController` edits `isUsedInConjunctionWithCarPlayWindow` directly or something.

It's a minor thing, so take it or leave it.